mungegithub/mungers/testowner/owner.go

```diff
@@ -49,10 +50,31 @@ type OwnerList struct {
 // or else the empty string if none is found.
 func (o *OwnerList) TestOwner(testName string) string {
 	name := normalize(testName)
+
+	// exact mapping
 	owner, _ := o.mapping[name]
+
+	// glob matching
+	if owner == "" {
+		for k, v := range o.mapping {
```

Comment:
add a note that this will iterate patterns in random order, so if there are overlapping matches we will have nondeterministic owners